examples/ts-code-mode-web/src/lib/reports/use-persisted-reports.ts

```
@@ -1,6 +1,6 @@
 'use client'
 
-import { useState, useEffect, useCallback, useRef } from 'react'
+import { useState, useEffect, useCallback } from 'react'
```

ESLint

[error] Member 'useEffect' of the import declaration should be sorted alphabetically.

(sort-imports)